lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb

```diff
@@ -88,9 +89,23 @@ def check_task_status(task_id)
 
     def mark_for_retry(task_id)
       @workspace.root['ae_result'] = 'retry'
-      @workspace.root['ae_retry_interval'] = 1.minute
+      @workspace.root['ae_retry_interval'] = retry_interval.minutes
       @workspace.persist_state_hash['automate_workspace_guid'] = @aw.guid
       @workspace.persist_state_hash[method_key] = task_id
+      $miq_ae_logger.info("Setting retry interval: #{@workspace.root['ae_retry_interval']}")
```

Comment:
@tinaafitz 
Should this message indicate that this is the internal retry value as opposed to the value set from the playbook.